### PR TITLE
Clean up verkv

### PR DESCRIPTION
**Pull request by zhongzc**
Signed-off-by: Zhenchi <zhongzc_arch@outlook.com>

Issues: https://github.com/tikv/tikv/issues/10002

### CODE REVIEW

**Comment by zhongzc** [comment]
cc @sticnarf

**Review by andylokandy — APPROVED** [review]

**Review by ekexium — APPROVED** [review]

File: proto/tikvpb.proto [hunk 1]
```diff
@@ -146,12 +138,8 @@ message BatchCommandsRequest {
             kvrpcpb.CheckTxnStatusRequest CheckTxnStatus = 25;
             kvrpcpb.TxnHeartBeatRequest TxnHeartBeat = 26;
 
-            kvrpcpb.VerGetRequest VerGet = 27;
```
Comment (tisonkun) [inline proto/tikvpb.proto:149]: The original `VerGetRequest` definition is still there, what is the plan to deprecate or remove?

Comment (zhongzc) [reply proto/tikvpb.proto:149]: My mistakes. I will remove them.

Comment (zhongzc) [reply proto/tikvpb.proto:149]: Addressed at https://github.com/pingcap/kvproto/pull/754

PTAL @andylokandy @ekexium

File: proto/tikvpb.proto [hunk 2]
```diff
@@ -146,12 +138,8 @@ message BatchCommandsRequest {
...
-            kvrpcpb.VerBatchGetRequest VerBatchGet = 28;
-            kvrpcpb.VerMutRequest VerMut = 29;
-            kvrpcpb.VerBatchMutRequest VerBatchMut = 30;
-            kvrpcpb.VerScanRequest VerScan = 31;
-            kvrpcpb.VerDeleteRangeRequest VerDeleteRange = 32;
+            // Deprecated for VerKV
+            // reserved 27 to 32;
```
Comment (BusyJay) [inline proto/tikvpb.proto:142]: Why comment it?

Comment (zhongzc) [reply proto/tikvpb.proto:142]: `reserved` is just valid for `message` and unsupported for `oneof`. Protoc cannot understand the comment, but at least this comment can remind humans.

Comment (BusyJay) [reply proto/tikvpb.proto:142]: You can put it right after declaration of `oneof` inside the same message.

Comment (zhongzc) [reply proto/tikvpb.proto:142]: Interesting... Refresh my knowledge. I will handle it soon.

**Pull request merged**